packages/react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -762,7 +762,7 @@ export class SelectionZone extends React.Component<ISelectionZoneProps, ISelecti
   }
 
   private _isInputElement(element: HTMLElement): boolean {
-    return element.tagName === 'INPUT' || element.tagName === 'TEXTAREA';
+    return element.tagName === 'INPUT' || element.tagName === 'TEXTAREA' || element.getAttribute('contenteditable') === 'true';
```

Comment:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable) empty string should also be considered `contenteditable="true"`. To handle `contenteditable` I think this needs to be updated to

```suggestion
    return element.tagName === 'INPUT' || element.tagName === 'TEXTAREA' || element.getAttribute('contenteditable') === 'true' || element.getAttribute('contenteditable') === '';
```